projects/permissions/src/tokens/permissions.ts

```diff
@@ -0,0 +1,9 @@
+import {inject, InjectionToken} from '@angular/core';
+import {NAVIGATOR} from '@ng-web-apis/common';
+
+export const PERMISSIONS = new InjectionToken<Permissions>(
```

Comment:
I wonder if we should add `null` as possible type for unsupporting browsers, or is it fine as is, because we have SUPPORT token. What do you think, @MarsiBarsi ?